src/Database/Query.php

```diff
@@ -2168,60 +2160,6 @@ public function setValueBinder(?ValueBinder $binder)
         return $this;
     }
 
-    /**
-     * Enables/Disables buffered results.
-     *
-     * When enabled the results returned by this Query will be
-     * buffered. This enables you to iterate a result set multiple times, or
-     * both cache and iterate it.
-     *
-     * When disabled it will consume less memory as fetched results are not
-     * remembered for future iterations.
-     *
-     * @param bool $enable Whether to enable buffering
-     * @return $this
-     */
-    public function enableBufferedResults(bool $enable = true)
```

Comment:
These methods look like good candidates for deprecation warnings in 4.x

Reply:
I have added a point to #16267 regarding adding deprecation to 4.x once we are doing with the changes in 5.0.